### PR TITLE
Fix samples link in XML documentation.

### PR DESCRIPTION
**Pull request by codemzs**
towards #3474

### CODE REVIEW

File: src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs [hunk 1]
```diff
@@ -643,7 +643,7 @@ public static LbfgsPoissonRegressionTrainer LbfgsPoissonRegression(this Regressi
         /// <example>
         /// <format type="text/markdown">
         /// <![CDATA[
-        ///  [!code-csharp[PoissonRegression](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/LbfgsPoissonMaximumEntropy.cs)]
+        ///  [!code-csharp[PoissonRegression](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/LbfgsMaximumEntropy.cs)]
```
Comment (shmoradims) [inline src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs]: >PoissonRegression [](start = 27, length = 17)

nit: Lbfgs

File: src/Microsoft.ML.TensorFlow/TensorFlowModel.cs [hunk 2]
```diff
@@ -56,13 +56,6 @@ public DataViewSchema GetInputSchema()
         /// <param name="outputColumnName">The name of the requested model output.</param>
         /// <param name="addBatchDimensionInput">Add a batch dimension to the input e.g. input = [224, 224, 3] => [-1, 224, 224, 3].
         /// This parameter is used to deal with models that have unknown shape but the internal operators in the model require data to have batch dimension as well.</param>
-        /// <example>
-        /// <format type="text/markdown">
-        /// <![CDATA[
-        /// [!code-csharp[ScoreTensorFlowModel](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlowTransform.cs)]
-        /// ]]>
-        /// </format>
-        /// </example>
```
Comment (shmoradims) [inline src/Microsoft.ML.TensorFlow/TensorFlowModel.cs]: please check with ZeeshanA. TextClassification.cs seems to be the correct sample for this API.

File: src/Microsoft.ML.TensorFlow/TensorFlowModel.cs [hunk 3]
```diff
@@ -56,13 +56,6 @@ public DataViewSchema GetInputSchema()
         /// <param name="outputColumnName">The name of the requested model output.</param>
         /// <param name="addBatchDimensionInput">Add a batch dimension to the input e.g. input = [224, 224, 3] => [-1, 224, 224, 3].
         /// This parameter is used to deal with models that have unknown shape but the internal operators in the model require data to have batch dimension as well.</param>
-        /// <example>
-        /// <format type="text/markdown">
-        /// <![CDATA[
-        /// [!code-csharp[ScoreTensorFlowModel](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlowTransform.cs)]
```
Comment (natke) [inline src/Microsoft.ML.TensorFlow/TensorFlowModel.cs]: We don't have this sample?

File: src/Microsoft.ML.TensorFlow/TensorFlowModel.cs (hunk 2 above)
Comment (shmoradims) [reply src/Microsoft.ML.TensorFlow/TensorFlowModel.cs]: /cc @zeahmed

Comment (codemzs) [reply src/Microsoft.ML.TensorFlow/TensorFlowModel.cs]: Yes but it is more array of input/output column and this API is just for single input/output column pair.

File: src/Microsoft.ML.TensorFlow/TensorFlowModel.cs (hunk 3 above)
Comment (codemzs) [reply src/Microsoft.ML.TensorFlow/TensorFlowModel.cs]: We didn't but I just modified an existing sample so that we can have a sample for this API :)

**Pull request merged**